### PR TITLE
Refactor/new sidebar

### PR DESCRIPTION
**Pull request by qwerg**
PR with various changes to get closer to the original UI UX project
BE4-cms-MR

### CODE REVIEW

File: src/Template/Pages/Dashboard/index.twig [hunk 1]
```diff
@@ -40,25 +49,21 @@
                     { '_name': 'modules:list', 'object_type': 'objects' },
                     { 'title': __('All objects'), 'class': 'dashboard-item has-background-module-objects icon-th-large-1', } )|raw }}
 
-                {# <a href="#" class="dashboard-item has-background-black icon-list-1" disabled>{{ __('All categories') }}</a> #}
+                <a href="#" class="dashboard-item has-background-module-categories icon-list-1">{{ __('All categories') }}</a>
```
Comment (stefanorosanelli) [inline src/Template/Pages/Dashboard/index.twig:53]: `All categories` and `All tags` are not available yet - better keep both anchors commented

Comment (stefanorosanelli) [reply src/Template/Pages/Dashboard/index.twig:53]: Note: we have `Categories` under `Data modeling` and it's ok to handle them there.

If we want to have a standalone `All categories` module to be used by non-admin users we will probably clone it or (better) prepare two possible views.

**Review by stefanorosanelli — CHANGES_REQUESTED** [review]
there's a layout problem in modules index when some custom filters are defined 

New version
image


Previous version (however not optimal)
image

File: src/Template/Pages/Dashboard/index.twig (hunk 1 above)
Comment (qwerg) [reply src/Template/Pages/Dashboard/index.twig:53]: Ok, I agree on a specific `Categories` view for non-admin, and keeping both anchors commented for now.

**Review by stefanorosanelli — APPROVED** [review]

**Pull request merged**